### ISSUE TITLE
Blockbase: Color schemes in customizer are broken

### ISSUE THREAD

**Issue by sublimese** [Automattic/themes#5001]
### Steps to replicate

1. Switch to Blockbase
2. Open the customizer
3. Select one of the non-default color schemes
4. Press publish
5. Notice an error message appears:  Looks like something’s gone wrong. Wait a couple seconds, and then try again.

## Result

Error message on screen:
Looks like something’s gone wrong. Wait a couple seconds, and then try again.

Error message in logs:
PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($function) must be a valid callback, non-static method GlobalStylesColorPalettes::sanitize_color_palette() cannot be called statically in C:\inetpub\wwwroot\someSite\wp-includes\class-wp-hook.php:303

## Expected

Color scheme change successfully

* User report: [shortlink to chat/forum thread/ticket, ie 12345-hc]
* Follow-up ticket: [shortlink to ticket, ie 12345-zen]

**Comment by pbking** [comment]
I was not able to reproduce this issue (on /trunk as of today) in either .com or .org environment.  However while testing I did find and fix [something similar](https://github.com/Automattic/themes/pull/5014).  Not likely related but maybe worth mentioning.

**Comment by sublimese** [comment]
I am happy to test something out if you would like.  Can also do screen grabs of each step that I am doing to get the issue.

**Comment by scruffian** [comment]
> I am happy to test something out if you would like. Can also do screen grabs of each step that I am doing to get the issue.

Yes please!

**Comment by sublimese** [comment]
WP Version:
![Screen Shot 2021-11-19 at 1 40 36 PM](https://user-images.githubusercontent.com/47400870/142689707-f473c671-9804-432a-bd06-77de6d91faa7.png)

Blockbase Version:
![Screen Shot 2021-11-19 at 1 40 48 PM](https://user-images.githubusercontent.com/47400870/142689743-c9272c12-b1d7-4860-9081-c8ea8a838de2.png)

Gutenberg (And all plugins)
![Screen Shot 2021-11-19 at 1 41 01 PM](https://user-images.githubusercontent.com/47400870/142689820-0f987f0d-6709-4469-b052-e74309361db1.png)

PHP Version:
![Screen Shot 2021-11-19 at 1 41 57 PM](https://user-images.githubusercontent.com/47400870/142689839-76b7981c-8a9a-4405-9f88-eaeedee5f56b.png)

Pages on Site:
![Screen Shot 2021-11-19 at 1 42 25 PM](https://user-images.githubusercontent.com/47400870/142689866-0e7a2406-e257-40af-988c-82a62c65339e.png)

All themes (showing Blockbase as active):
![Screen Shot 2021-11-19 at 1 42 54 PM](https://user-images.githubusercontent.com/47400870/142689965-77fd90fa-7c1f-4202-9dd8-1fcb6bfa2a03.png)

Screen after clicking "customize" on the previous screen:
![Screen Shot 2021-11-19 at 1 43 05 PM](https://user-images.githubusercontent.com/47400870/142690137-32676e34-5cb8-4dcc-90e2-19e7567bf869.png)

Selecting "Colors" on previous screen:
![Screen Shot 2021-11-19 at 1 43 16 PM](https://user-images.githubusercontent.com/47400870/142690209-c1045e29-8158-4caf-ab70-e459faf204aa.png)

Selecting one of the "Featured" Color Scheme's at the bottom of the screen (I picked the one with red, but it happens on all):
![Screen Shot 2021-11-19 at 1 43 27 PM](https://user-images.githubusercontent.com/47400870/142690310-4513057e-f3a0-4fb8-9821-5040e7bac25e.png)

Hit publish button:
![Screen Shot 2021-11-19 at 1 43 35 PM](https://user-images.githubusercontent.com/47400870/142690344-08cd1f44-18fc-4003-b1fe-b6d5a954ad73.png)

Screen shot of error generated in log:
![Screen Shot 2021-11-19 at 1 45 57 PM](https://user-images.githubusercontent.com/47400870/142690378-eb1276ce-4577-49f2-a369-fbddc7474bc5.png)



As you can see this is a VERY simple site.  There is almost nothing on it.  Error is very easy to generate.

Please let me know if you need\want me to do anything else.

Thanks!

**Comment by matiasbenedetto** [comment]
I was able to reproduce this error running **PHP 8**, it works ok with **PHP 7.4**

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/1310626/143091871-b3f3c6fb-f00c-4a0c-a1cb-d42ab3947fe5.png">


```
wordpress_1  | [Tue Nov 23 19:38:56.040698 2021] [php:error] [pid 70] [client 172.20.0.1:51308] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method GlobalStylesColorPalettes::sanitize_color_palette() cannot be called statically in /var/www/html/wp-includes/class-wp-hook.php:303\nStack trace:\n#0 /var/www/html/wp-includes/plugin.php(189): WP_Hook->apply_filters('palette-1', Array)\n#1 /var/www/html/wp-includes/class-wp-customize-setting.php(575): apply_filters('customize_sanit...', 'palette-1', Object(WP_Customize_Setting))\n#2 /var/www/html/wp-includes/class-wp-customize-manager.php(2367): WP_Customize_Setting->sanitize('palette-1')\n#3 /var/www/html/wp-includes/class-wp-customize-manager.php(2775): WP_Customize_Manager->validate_setting_values(Array, Array)\n#4 /var/www/html/wp-includes/class-wp-customize-manager.php(2522): WP_Customize_Manager->save_changeset_post(Array)\n#5 /var/www/html/wp-includes/class-wp-hook.php(303): WP_Customize_Manager->save('')\n#6 /var/www/html/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)\n#7 /var/www/html/wp-includes/plugin.php(470): WP_Hook->do_action(Array)\n#8 /var/www/html/wp-admin/admin-ajax.php(187): do_action('wp_ajax_customi...')\n#9 {main}\n  thrown in /var/www/html/wp-includes/class-wp-hook.php on line 303, referer: http://localhost:8282/wp-admin/customize.php?return=%2Fwp-admin%2Fthemes.php
```

**Comment by scruffian** [comment]
This should be fixed by #5082

**Comment by archcort** [comment]
Hello there, I'm experiencing the same problem even with php8.0. Can't save changes in the color scheme.
<img width="1008" alt="Screenshot 2021-12-29 at 22 28 52" src="https://user-images.githubusercontent.com/7091354/147696977-ff1effa1-b6d7-4229-9bd1-57808ec39333.png">
